src/bors/handlers/review.rs

```diff
@@ -127,14 +132,50 @@ pub(super) async fn command_unapprove(
     ) {
         repo_state
             .client
-            .post_comment(pr.number(), unapprove_non_open_pr_comment())
+            .post_comment(pr_num, unapprove_non_open_pr_comment())
             .await?;
         return Ok(());
     }
 
     db.unapprove(&pr.db).await?;
     handle_label_trigger(&repo_state, pr.number(), LabelTrigger::Unapproved).await?;
-    notify_of_unapproval(&repo_state, pr).await
+    notify_of_unapproval(&repo_state, pr).await?;
```

Comment:
This will still send two comments, right? What I meant here was to essentially try to cancel the workflows first, store the result into a string, and optionally include that string in the unapproval comment.